### DISCUSSION TITLE
Using this plugin adds nearly 300KB to build

### DISCUSSION THREAD

**Discussion by craigrileyuk** [sambauers/tailwindcss-3d#450] [Q&A]
Love the extra functionality, but I think something must be misconfigured somewhere since the only difference in the below production builds is the inclusion of this Tailwind plugin:

- public/build/assets/app-gzJ6WKzs.js - 669.23 kB
- public/build/assets/app-_nUAZWyP.js - 933.96 kB

I've only used a couple of classes, but it seems to have included everything in CSS-in-JS format inside my build.

**Comment by sambauers** [comment]
Are you installing this plugin as a development dependency?

It shouldn’t be part of your build at all.

Some reproduction steps for your issue may help here.
